tsl/src/compression/api.c

```diff
@@ -1190,6 +1198,10 @@ recompress_chunk_segmentwise_impl(Chunk *uncompressed_chunk)
 
 	/* Index scan */
 	Relation index_rel = index_open(row_compressor.index_oid, ExclusiveLock);
+	ereport(LOG,
+			(errmsg("locks acquired for recompression: \"%s.%s\"",
+					NameStr(uncompressed_chunk->fd.schema_name),
+					NameStr(uncompressed_chunk->fd.table_name))));
```

Comment:
Same here, having the lock name in the log message will make it easier to debug, especially of the request and the acquire is far apart and you have other locks mentioned in the log.